Tasks/Gradle/CodeAnalysis/Common/CheckstyleTool.ts

```diff
@@ -26,6 +26,25 @@ export class CheckstyleTool extends BaseTool {
         super('Checkstyle', buildOutput, boolInputName);
     }
 
+    public configureBuild(toolRunner: ToolRunner): ToolRunner {
+        if (this.isEnabled()) {
+            console.log(tl.loc('codeAnalysis_ToolIsEnabled'), this.toolName);
+
+            switch (this.buildOutput.buildEngine) {
+                case BuildEngine.Maven: {
+                    toolRunner.arg(['checkstyle:checkstyle']);
```

Comment:
The Maven Checkstyle plugin uses the Sun rules by default, so the behaviour is identical to Gradle (except for version differences).